core/src/main/scala/tech/beshu/ror/accesscontrol/blocks/variables/runtime/RuntimeResolvableGroupsLogic.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.accesscontrol.blocks.variables.runtime
+
+import cats.data.NonEmptyList
+import tech.beshu.ror.accesscontrol.blocks.BlockContext
+import tech.beshu.ror.accesscontrol.domain.{GroupIdLike, GroupIds, GroupsLogic}
+import tech.beshu.ror.accesscontrol.utils.RuntimeMultiResolvableVariableOps.resolveAll
+import tech.beshu.ror.utils.uniquelist.UniqueNonEmptyList
+
+case class RuntimeResolvableGroupsLogic[GL <: GroupsLogic](groupIds: UniqueNonEmptyList[RuntimeMultiResolvableVariable[GroupIdLike]],
```

Comment:
it's not a data class because of the function `GroupIds => GL` as a param

Reply:
I made it a case class in order to provide equals method. I noticed it too and reverted comparing instances of this class in tests, but forgot to revert making it  `case`. Reverted.